simulator/README.md

```diff
@@ -0,0 +1,161 @@
+# trace-to-events
+
+## Introduction
+
+## Command Line Interface
+
+```shell
+simulator [OPTIONS] --broker <BROKER> [COMMAND]
+```
+
+For instance:
+
+```shell
+simulator --broker localhost:19092 \
+    --trace-topic Traces \
+    --event-topic SimulatedEvents \
+    json \
+      --path "trace.json" \
+      --repeat=10
+```
+
+For instructions run:
+
+```shell
+simulator --help
+```
+
+### Commands
+
+- `Single`:            Produce a single trace.
+- `Continuous`:        Produce a regular infinite sequence of traces are regular intervals.
+- `Json`:              Produce traces in the manner specified by a json file.
+
+## Json Format
+
+In `json` mode, the behavior is given by the simulator object in the user-defined json file.
+The structure of the simulator object is:
+
+### Simulator
+
+- voltage: `Interval`
```

Comment:
Could you add relative links to these types to make navigating this on the GitHub render easier.